### DISCUSSION TITLE
Issue creating new user

### DISCUSSION THREAD

**Discussion by johnathanmulhall** [Budibase/budibase#7446] [Help] [answered]
I am testing adding users. I have added a new user and generated the password for them. However, they are not added to any apps. Currently the only user is my admin.

**Comment by R2bEEaton** [comment, marked as answer]
Hi @johnathanmulhall!

The users are probably in there, but not assigned to an app. In the Apps screen from the portal, click Manage on the desired app.
![image](https://user-images.githubusercontent.com/34921506/186528864-95ecce2c-ba2a-46a5-88da-853326ab6e16.png)

Then, go to the Access tab. Click "Assign Users" then use the text box to search for your added user and the dropdown to set the desired app-specific access level. You can do this in bulk by clicking "Add Email." When you've set them all, click "Done" and the user should now have access to that app.

The documentation found [here](https://docs.budibase.com/docs/user-roles) is currently out of date.

Reply with any questions, otherwise if this works for you, make sure to accept this answer using the check box above so as to help others in the future.

**Reply by johnathanmulhall** [reply]
Thank you so much. I forgot about this step.

Appreciate your help.
